AABB_tree/include/CGAL/AABB_tree.h

```diff
@@ -581,10 +603,15 @@ namespace CGAL {
     }
 
     // search KD-tree
-    mutable const Search_tree* m_p_search_tree;
-    mutable bool m_search_tree_constructed;
-    mutable bool m_default_search_tree_constructed; // indicates whether the internal kd-tree should be built
-    bool m_need_build;
+    const Search_tree* m_p_search_tree = nullptr;
+    bool m_use_default_search_tree = true; // indicates whether the internal kd-tree should be built
+#ifdef CGAL_HAS_THREADS
+    std::atomic<bool> m_atomic_need_build;
+    std::atomic<bool> m_atomic_search_tree_constructed;
+#else
+    bool m_need_build = false;
+    mutable bool m_search_tree_constructed = false;
```

Comment:
This variable probably no longer needs the `mutable` keyword.

Reply:
It does, otherwise is does not compile when threads is OFF (it is used in a const function while with threads it is a local variable).